tests/config/run_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -xe
+
+if [ $# -eq 0 ]; then
+  echo "Error: Please append path to the test binary path as a command-line argument."
+  exit 1
+fi
+
+TEMP_DIR="$(mktemp -d)"
+pushd "${TEMP_DIR}"
+
+function clean() {
+    rm -r "${TEMP_DIR}"
+}
+trap clean EXIT
+
+# Assign the base path from the argument
+binary_path="$1"
+
+cp "${binary_path}/clvk.conf" "${TEMP_DIR}"
+
+# Assing an env var for testing
+default_color_val=$(echo CLVK_LOG_COLOUR)
+export CLVK_LOG_COLOUR=1
```

Comment:
Do not export, just define it as `CLVK_CONFIG_FILE` when running the test.

Reply:
This is not resolved, I still see the export. Just define `CLVK_LOG_COLOUR=1` as the test does for `CLVK_CONFIG_FILE`. That way, the env variable is not exported, it is just set for the execution of `config_test` and does not need to be restored after the run.